### PR TITLE
Update directory structure

### PR DESCRIPTION
**Pull request by bigfoot90**

### CODE REVIEW

**Comment by bigfoot90** [comment]
I dont know why travis is failing, locally the tests are green.

File: tests/_data/simple_multipart.txt [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-------------------------------83ff53821b7c
-Content-Disposition: form-data; name="foo"
-
-bar
-------------------------------83ff53821b7c
-Content-Disposition: form-data; name="rfc5987"; text1*=iso-8859-1'en'%A3%20rates; text2*=UTF-8''%c2%a3%20and%20%e2%82%ac%20rates
-
-rfc
-------------------------------83ff53821b7c--
+User-Agent: curl/7.21.2 (x86_64-apple-darwin)
```
Comment (goetas) [inline tests/_data/simple_multipart.txt]: this files seems changed. The encoding here was important.

Comment (goetas) [reply tests/_data/simple_multipart.txt]: probably thats why is failing

Comment (bigfoot90) [reply tests/_data/simple_multipart.txt]: Fixed. Previously I renamed directory  by ide, now renamed by `mv` terminal command.
It should be ok now.

**Pull request merged**